features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtilsTest.kt

```diff
@@ -379,14 +373,13 @@ internal class DrawableUtilsTest {
     }
 
     @Test
-    fun `M not use original drawable W createBitmapOfApproxSizeFromDrawable`() {
+    fun `M uses original drawable W createBitmapOfApproxSizeFromDrawable`() {
```

Comment:
It's not the original drawable since we now rely on the drawable being copied previously. I'm not sure this test makes sense to keep